Issues/534

qclin opened the pull request:
fixes #534 

new metadata.areas list replaces Country & region
bind view to model

matthewberryman commented:
Note I checked in prod database and there are 4 events that have metadata.region specified that I can manually re-enter, i.e. safe for you to ignore that field but you do need to handle missing .areas properly.

qclin commented:
added handling on events & eventsID page for older entry without areas field
es-lint update loop variables 
assignAreas() > function already removed since data-model mapping 
cleaned console.log

matthewberryman commented:
@qclin I've left the branch around as it may be easier to merge that into issues/478 to resolve the merge conflicts in https://github.com/MSFREACH/msf-reach/pull/564
